doc/man/scons.xml

```diff
@@ -3922,61 +3917,79 @@ char function_name();
 </programlisting>
 
 <para>
-Returns an empty string on success, a string containing
-an error message on failure.
+Note: do not use <parameter>header</parameter>
+to include the standard header file that declares
+<parameter>function_name</parameter> - successful
+compilation of the test program depends on using
+a dummy prototype for it,
+to avoid probems with compilers which object to
+function signature mismatches.
 </para>
+
+<para>Returns a boolean indicating success or failure.</para>
```

Comment:
The CheckFunc() does return a string on failure, and True value for success. 
Or at least if it fails when calling _lang2suffix()

Reply:
Nevermind. Was looking at ConfTest, not SConf.

Reply:
That's the underlying implementation in `Confstr.py`.  The reason there's an incorrect statement in previous manpage (as opposed to no comment at all, like it was back in the 3.x days and before) is I got confused by the two-layer stuff. This time, I went through and made sure the Check methods in `SConf.py` that say `return not res` (which is all but one) are marked as returning a bool. Fun. (yes, I know you already closed and merged, just wanted to add)